quilt/tools/command.py

```diff
@@ -581,6 +587,7 @@ def main():
     build_p.add_argument("package", type=str, help="Owner/Package Name")
     buildpath_group = build_p.add_mutually_exclusive_group(required=True)
     buildpath_group.add_argument("-d", "--directory", type=str, help="Source file directory")
```

Comment:
Agree `quilt generate -d DIRPATH` And error if it will clobber an existing build for